### ISSUE TITLE
Expand page title tag descriptions

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#4743]
Observing a screen reader user, I noticed many of our page titles provide insufficient information about the page content. 

Here's an example:

![Browser tab with a page title of Downloads](https://user-images.githubusercontent.com/32855580/67511938-15ce3780-f655-11e9-95bf-dffb628dee0f.png)

When this page title is read aloud, it simply says "Downloads," which can be too limiting if the user has multiple tabs open and is navigating between them. 

We should, at the very least, append or prepend "Natural Resources Revenue Data."
